### PR TITLE
Added standardized TimeSpan string parser

### PR DESCRIPTION
**Pull request by IAmSilK**
Also added TimeSpan parameter resolution using added parser

Examples of unparsed inputs:
- 20 seconds
- 30 days, 40 minutes, and 50 seconds
- 10h20m30s
- 3.5 days 30 seconds
- 1234.123     milliseconds

### CODE REVIEW

File: framework/OpenMod.Core/Commands/Parameters/TimeSpanCommandParameterResolveProvider.cs [hunk 1]
```diff
@@ -0,0 +1,25 @@
+﻿using OpenMod.API.Commands;
+using OpenMod.Core.Helpers;
+using System;
+using System.Threading.Tasks;
+
+namespace OpenMod.Core.Commands.Parameters
+{
+    public class TimeSpanCommandParameterResolveProvider : ICommandParameterResolveProvider
```
Comment (CyberAndrii) [inline framework/OpenMod.Core/Commands/Parameters/TimeSpanCommandParameterResolveProvider.cs:8]: You forgot to add `[Priority(Priority = Priority.Low)]` attribute.

**Review by CyberAndrii — CHANGES_REQUESTED** [review]

File: framework/OpenMod.Core/Commands/Parameters/TimeSpanCommandParameterResolveProvider.cs (hunk 1 above)
Comment (IAmSilK) [reply framework/OpenMod.Core/Commands/Parameters/TimeSpanCommandParameterResolveProvider.cs:10]: Added

**Comment by CyberAndrii** [comment]
Also better use library from #100

**Comment by Trojaner** [comment]
@CyberAndrii I think this is good enough. But for DateTime parsing we need support for specifying Date directly, e.g. 12.01.2020 13:23 or "1 week later"

**Comment by IAmSilK** [comment]
Library stated in #100 is for DateTime where as this is for a difference in dates - TimeSpan

**Pull request merged**